commands/motivation.js

```diff
@@ -0,0 +1,21 @@
+const { SlashCommandBuilder } = require("discord.js");
+const axios = require("axios").default;
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName("motivation")
+    .setDescription("Get a random motivation quote."),
+  async execute(interaction) {
+    await axios({
+      method: "get",
+      url: "https://nodejs-quoteapp.herokuapp.com/quote",
+      responseType: "json",
+    })
+      .then((response) => {
+        interaction.reply("``"+response.data.quote+"``");
```

Comment:
@komen205 Almost 😄 There need to be 3x ` on each side for discord to know that this is a code block :)

Reply:
Done @thieleju